src/service/action-impl.js

```diff
@@ -361,9 +361,9 @@ export class ActionService {
   /**
    * @param {!Element} node
    * @param {string} actionEventType
-   * @return {?ActionInfoDef}
+   * @return {?Array<ActionInfoDef>}
```

Comment:
here it does make sense to keep it nullable to indicate match or not. so no change here.